src/noun.substance

```diff
@@ -469,6 +469,26 @@ hs: limestone
 hyper: calcium_ion
 hypo: metal
 
+w: calciarenite
+d: A limestone consisting predominantly (> 50%) of rounded detrital calcite particles (either bigenetic or non-biogenetic) between 0.063 and 2 mm. Interstices are filled with carbonate mud or cement (Füchtbauer, 1988; Neuendorf et al., 2005).
```

Comment:
Can we simplify that? What about:

> A limestone consisting predominantly (> 50%) of rounded detrital calcite particles between 0.063 and 2 mm (Füchtbauer, 1988; Neuendorf et al., 2005).

Reply:
We could, but then we’d loose the information about the interstices, which IMHO is quite relevant. I don’t see much prejudice on removing the biogenetic/non-bio however. Could we meet halfway in:
A limestone consisting predominantly (> 50%) of rounded detrital calcite particles between 0.063 and 2 mm. Interstices are filled with carbonate mud or cement (adapted from Füchtbauer, 1988; Neuendorf et al., 2005).“

Reply:
OK, vide email que mandei agora.